go/src/koding/kites/kloud/team/mongo.go

```diff
@@ -87,11 +85,9 @@ func groups2teams(groups ...*models.Group) []*Team {
 			continue
 		}
 
-		members, _ := group.Counts["members"].(int)
```

Comment:
Could you please also rm two things:

1) `Members` field from `Team` [here](https://github.com/koding/koding/blob/e19251e0ef2b53c1a485ac62aa8a9f6bfd09e67f/go/src/koding/kites/kloud/team/client.go#L9) 

2) `MEMBERS` column [here](https://github.com/koding/koding/blob/e19251e0ef2b53c1a485ac62aa8a9f6bfd09e67f/go/src/koding/kites/kloud/team/client.go#L9)

Seems like I forgot to do this before because we were talking about this..